lib/ohai/plugins/cloud.rb

```diff
@@ -189,7 +189,7 @@ def get_openstack_values
     cloud[:public_hostname] = openstack["public_hostname"]
     cloud[:local_ipv4] = openstack["local_ipv4"]
     cloud[:local_hostname] = openstack["local_hostname"]
-    cloud[:provider] = openstack["provider"]
+    cloud[:provider] = "openstack"
```

Comment:
The previous behavior was HP or Openstack, but I'll switch this one back.  People on Dreamhost can figure it out probably.